tests/mesh/mesh_generation_test.C

```diff
@@ -86,10 +86,17 @@ public:
   void testBuildLine(UnstructuredMesh & mesh, unsigned int n, ElemType type)
   {
     MeshTools::Generation::build_line (mesh, n, -1.0, 2.0, type);
+
     CPPUNIT_ASSERT_EQUAL(mesh.n_elem(), cast_int<dof_id_type>(n));
     CPPUNIT_ASSERT_EQUAL(mesh.n_nodes(),
                          cast_int<dof_id_type>((Elem::type_to_n_nodes_map[type]-1)*n + 1));
 
+   // Make sure that subdomain ids for unpartitioned elements are included
+   std::set<subdomain_id_type> ids;
+   mesh.subdomain_ids(ids);
+   CPPUNIT_ASSERT_EQUAL(ids.size(), (std::size_t)1);
+   CPPUNIT_ASSERT_EQUAL(*ids.begin(), (subdomain_id_type)0;
```

Comment:
This line (and a similar one below) is missing a trailing parenthesis?

Reply:
Yeah - I pushed those in haste before testing and actually removed. 

I thought I could get away with testing this with the MeshTools::Generation meshes, but I forgot that we call prepare_for_use() at the end of all of those calls. Therefore, I wouldn't really be testing as we'd partition.